shared/src/main/java/net/blay09/mods/waystones/client/ModClientEventHandlers.java

```diff
@@ -2,10 +2,23 @@
 
 import net.blay09.mods.balm.api.Balm;
 import net.blay09.mods.balm.api.event.client.FovUpdateEvent;
+import net.blay09.mods.waystones.api.KnownWaystonesEvent;
+import net.blay09.mods.waystones.api.WaystoneUpdateReceivedEvent;
+import net.blay09.mods.waystones.compat.JourneyMapAddon;
 import net.blay09.mods.waystones.handler.WarpStoneFOVHandler;
 
 public class ModClientEventHandlers {
     public static void initialize() {
         Balm.getEvents().onEvent(FovUpdateEvent.class, WarpStoneFOVHandler::onFOV);
+        Balm.getEvents().onEvent(KnownWaystonesEvent.class, (event)-> {
+            if(JourneyMapAddon.getInstance() != null) {
+                JourneyMapAddon.getInstance().onKnownWaystones(event);
+            }
+        });
+        Balm.getEvents().onEvent(WaystoneUpdateReceivedEvent.class, (event)-> {
+            if(JourneyMapAddon.getInstance() != null) {
+                JourneyMapAddon.getInstance().onWaystoneUpdateReceived(event);
+            }
+        });
```

Comment:
This looks like it will break if JourneyMap is not loaded as it will try to load the class containing JourneyMap references. Better to register these inside the JourneyMapAddon constructor.

Reply:
The null check should prevent that. 
But it does make more sense to register it in the addon class. 
`JourneyMapAddon.getInstance() != null)`